pyproject.toml

```diff
@@ -130,6 +130,7 @@ exclude = [
   '^music_assistant/providers/deezer/.*$',
   '^music_assistant/providers/dlna/.*$',
   '^music_assistant/providers/hass_players/.*$',
+  '^music_assistant/providers/nugs/.*$',
```

Comment:
We don't allow any new providers without mypy coverage (we still need to catch up with the existing ones).
Looking at the code its probably not far away from passing mypy checks so lets focus on fixing the remaining items.